sqlmesh/core/engine_adapter/base.py

```diff
@@ -321,6 +321,7 @@ def replace_query(
         table_name: TableName,
         query_or_df: QueryOrDF,
         columns_to_types: t.Optional[t.Dict[str, exp.DataType]] = None,
+        level: int = logging.DEBUG,
```

Comment:
It is not clear from the name that this log level is just for execution (right?). If so, can we call this `execute_log_level`?